.gitignore

```diff
@@ -1,7 +1,9 @@
 # Build files
 /target
-Cargo.lock
```

Comment:
You are probably already aware of this, but `cargo install` ignores lockfile when executed without `--locked`. Nothing actionable here, just thought I would note.

Reply:
Yeah, that inclusion is for git.